trajopt/include/trajopt/file_write_callback.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <trajopt/common.hpp>
+#include <trajopt/problem_description.hpp>
+#include <trajopt_utils/eigen_conversions.hpp>
+#include <tesseract_core/basic_kin.h>
+#include <tesseract_core/basic_env.h>
+#include <trajopt/plot_callback.hpp>
+#include <functional>
+#include <set>
+#include <fstream>
+#include <string>
+#include <stdio.h>
+#include <Eigen/Core>
+
+typedef std::shared_ptr<std::ofstream> ofstreamPtr;
+
+using namespace Eigen;
+using namespace util;
+using namespace std;
```

Comment:
Is this meant to be a publicly accessed header? If so, do not put using directives in it. You're effectively also making the decision to use the same namespaces on behalf of all of your clients. Put it in the source file.

Reply:
Definitely needs to be changed. Didn't understand the extent of what this did back then